server/shared/src/main/scala/org/http4s/server/middleware/GZip.scala

```diff
@@ -51,6 +53,11 @@ object GZip extends GZipPlatform {
       (contentType.get.mediaType == MediaType.application.`octet-stream`))
   }
 
+  def defaultIsZipped[F[_]](req: Request[F]): Boolean =
+    req.headers.get[`Content-Encoding`].map(_.contentCoding).exists { coding =>
+      coding == ContentCoding.gzip || coding == ContentCoding.`x-gzip`
+    }
```

Comment:
I understand why `isZippable` is configurable, but I wonder is there any other reasonable implementation of `isZipped`?